README.md

```diff
@@ -54,9 +54,9 @@ This project is licensed under the Apache License version 2.0 - see the [LICENSE
 
 ## checkstyle
```

Comment:
Better to write this heading in CAPS

Reply:
sorry, was to hasty. Will change this next time.